src/commons/database-operations.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { LoggerService } from '@tazama-lf/frms-coe-lib';
+import { Knex } from 'knex';
+
+@Injectable()
+export class DatabaseOperationsService {
+  constructor(
+    private readonly loggerService: LoggerService,
+    @Inject('KNEX') private readonly knex: Knex,
+  ) {}
+
+  async addAccount(accountId: string, tenantId: string): Promise<void> {
+    try {
+      await this.knex('accounts').insert({
+        accountId,
+        tenantId,
+        created: new Date().toISOString(),
+      });
+      this.loggerService.log(`Added account: ${accountId} for tenant: ${tenantId}`);
+    } catch (error) {
+      this.loggerService.error(`Failed to add account: ${String(error)}`);
```

Comment:
Try to throw meaningful exceptions, for example:
```suggestion
      if (error instanceof Error && error.message.includes('unique constraint')) {
        throw new BadRequestException(`Account ${accountId} already exists for tenant ${tenantId}`);
      }
```